api-server/api/controllers/security.ts

```diff
@@ -180,8 +180,14 @@ export const VerifyLogin = async (
             } else {
               const brokerKey = decoded['id'];
               const jolokia = securityStore.get(brokerKey);
-              res.locals.jolokia = jolokia;
-              next();
+              if (jolokia) {
+                res.locals.jolokia = jolokia;
+                next();
+              } else {
+                res.status(401).json({
+                  message: 'This session has expired. Please login again',
```

Comment:
Hello @lavocatt , LGTM, just small suggestion here. instead of adding direct text here could you please update this with using `useTranslation` hook?
`message: {t('the_session_expired_login_again')}`
or-else any best message you think.

Reply:
Do we use the translation service for the `api-server` ?

Reply:
yes you're correct. translation services are not commonly used in api-server. but, there are some scenarios where translation might be necessary in an API context, such as: showing Error Messages, and Response Data if it includes user-visible text.

Reply:
That makes sense. I'll make a patch to translate all the messages coming from the api-server.